pkg/tsdb/cloudmonitoring/cloudmonitoring_test.go

```diff
@@ -989,7 +990,7 @@ func TestCloudMonitoring(t *testing.T) {
 func loadTestFile(path string) (cloudMonitoringResponse, error) {
 	var data cloudMonitoringResponse
 
-	jsonBody, err := ioutil.ReadFile(path)
+	jsonBody, err := ioutil.ReadFile(filepath.Clean(path))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.